domain/modeldefaults/service/service.go

```diff
@@ -39,26 +42,68 @@ type ModelConfigProviderFunc func(string) (environs.ModelConfigProvider, error)
 
 // State is the model config state required by this service.
 type State interface {
-	// ConfigDefaults returns the default configuration values set in Juju.
-	ConfigDefaults(context.Context) map[string]any
+	// GetModelCloudDetails returns the cloud UUID and region for the given model.
+	// If the model is not found, an error specifying [modelerrors,NotFound] is returned.
```

Comment:
errors are implementation details, not for the interface.

Reply:
Errors very much are a documented part of the interface behaviour.